src/style.css

```diff
@@ -0,0 +1,126 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
```

Comment:
you could't  fix width to block, only for item.
You can create for example page block, and page__card could be its item.